src/Sulu/Bundle/SnippetBundle/Command/SnippetLocaleCopyCommand.php

```diff
@@ -97,7 +97,7 @@ public function execute(InputInterface $input, OutputInterface $output)
         $overwrite = $input->getOption('overwrite');
         $dryRun = $input->getOption('dry-run');
 
-        $this->session = $this->getContainer()->get('doctrine_phpcr')->getManager()->getPhpcrSession();
+        $this->session = $this->getContainer()->get('doctrine_phpcr.session');
```

Comment:
Actually we should inject the correct session here... But I think this command is not working correctly at the moment, since it looks to me as if the changes are not transferred to the live workspace with a `publish` call...